gnome/gtk3/Portfile

```diff
@@ -160,6 +160,18 @@ platform darwin {
     if {${os.major} <= 12} {
         # requires cups 1.7
         configure.args-append --disable-cups
+        # OS X 10.11 and earlier does not have NSEventTypeTabletProximity and the version check macro is broken
+        # see https://gitlab.gnome.org/GNOME/gtk/-/issues/3592
+        if {[variant_isset quartz]} {
+            patchfiles-append   patch-gdk_quartz-10.11_compat.diff
+        }
+    }
+    if {${os.major} <= 15} {
+        # OS X 10.11 and earlier does not have NSEventTypeTabletProximity and the version check macro is broken
+        # see https://gitlab.gnome.org/GNOME/gtk/-/issues/3592
+        if {[variant_isset quartz]} {
+            patchfiles-append   patch-gdk_quartz-10.11_compat.diff
+        }
```

Comment:
For OS X 10.8 and earlier, this will add the patchfile twice, which will fail.

Please just apply the patch once unconditionally on all systems and variants. This is simpler. Applying it conditionally on older systems is a good way to ensure that a problem occurs in the future when someone running a newer system updates the port and doesn't realize they have to adjust or remove the patch.

Reply:
Yes, you're right. I'll push a fix